Modules/SubModule_Population/xASL_adm_GzipAllFiles.m

```diff
@@ -58,14 +58,14 @@ function xASL_adm_GzipAllFiles(ROOT, bFolder, bUseLinux, pathExternal)
             if ~isempty(PathList)
                 fprintf('\n%s\n',['G-zZzZipping ' num2str(length(PathList)) ' files']);
                 numCores = feature('numcores');
+                glob_pat = join(['"' ROOT '/**/*.nii"']);
```

Comment:
We use camelCase with readable variable names, which enhances readability for others. 
Plus, note that you use a Unix slash / that won't work with a Windows system \\

So this should probably be:
if ispc
    globalPattern = ['"' ROOT '\**\*.nii"']);
else
    globalPattern = ['"' ROOT '/**/*.nii"']);
end

Reply:
It actually shouldn't care about forward slashs or backslashes. The glob library in Rust that this program calls is very robust in that regard. Example of mixing slashes in Windows:

![image](https://user-images.githubusercontent.com/57411571/216162219-1b4c2a56-5174-45c5-9497-084480395ca5.png)